ui/desktop/src/main.ts

```diff
@@ -1640,10 +1651,51 @@ const registerGlobalHotkey = (accelerator: string) => {
   }
 };
 
+// HTTP server for serving MCP proxy files
+let mcpUIProxyServerPort: number | null = null;
+let mcpUIProxyServer: http.Server | null = null;
+
+async function startMcpUIProxyServer(): Promise<number> {
+  return new Promise((resolve, reject) => {
+    const expressApp = express();
```

Comment:
nice - and this looks like it is already in the dependencies from electron?

Reply:
yeah! Express is already a dependency in the app.
https://github.com/block/goose/blob/feat/mcp-ui-improvements/ui/desktop/package.json#L70